src/ai/backend/manager/models/storage.py

```diff
@@ -199,14 +200,18 @@ class StorageVolume(graphene.ObjectType):
     class Meta:
         interfaces = (Item,)
 
-    # id: {proxy_name}:{name}
+    id = graphene.String()  # "{proxy}:{name}""
```

Comment:
@fregataa Please revert this line.
It occurs this error.
<img width="1341" alt="image" src="https://github.com/lablup/backend.ai/assets/28584164/63b4e74b-6cf0-4d96-83eb-ad88df8f86d0">

Reply:
I reverted this. but I am curious why it raises such error..

Reply:
I think it's TypeError. `Interface field Item.id expects type ID but StorageVolume.id is type String.`